tests/test_preprocessing.py

```diff
@@ -255,13 +255,13 @@ def check_scaling_features(use_feature_hashing=False, use_scaling=False):
 
     # these are the expected values of the f-measures, sorted
     if not use_feature_hashing:
-        expected_fmeasures = ([0.77319587628865982, 0.78640776699029125] if
+        expected_fmeasures = ([0.55276381909547745, 0.55721393034825872] if
                               not use_scaling else
-                              [0.94930875576036866, 0.93989071038251359])
+                              [0.65217391304347827, 0.70370370370370372])
```

Comment:
Okay, to double check that this is indeed the cause of the extreme changes, I generated the test data in a conda environment with scikit-learn 0.17.1 and serialized it to disk. Then in a second conda environment with scikit-learn v0.18.1, I loaded this serialized data and ran this specific test. The numbers match. So, the differences are all because of changes to `make_classification()`.